packages/variable-resolver/src/browser/variable-quick-open-service.ts

```diff
@@ -44,26 +63,15 @@ export class VariableQuickOpenService implements QuickOpenModel {
     onType(lookFor: string, acceptor: (items: QuickOpenItem[]) => void): void {
         acceptor(this.items);
     }
-}
-
-export class VariableQuickOpenItem extends QuickOpenItem {
```

Comment:
Should go to the breaking changes section.

Reply:
ok